### DISCUSSION TITLE
Welcome to the LPIC-2 book discussions!

### DISCUSSION THREAD

**Discussion by CoMPaTech** [lpic2book/src#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stappersg** [comment]
Hi, I'm _Stappers_, here is my account name @stappersg.  I have publish in my github.com profile how to contact me outside _GH_.  I'm a fan of https://git-send-email.io/  We will find out which common interrest we have.

**Comment by CoMPaTech** [comment]
Hi all, at [github.com/CoMPaTech](https://github.com/compatech) you can view my profile and interests. Looking forward to a growing community and better book for everyone. If someone is close to any of the LPIC board members, please reach out. At Sue (former Snow) we tried reaching out back in 2021 but unfortunately no responses from the board, as such it was generously donated to the commnunity here at [lpic2book](https://github.com/lpic2book).
